code/modules/cargo/packsrogue/bandit/sawbones.dm

```diff
@@ -3,50 +3,193 @@
+	name = "Studded Leather Armor"
+	cost = 50
+	time_lock = 45 MINUTES
+	contains = list(/obj/item/clothing/suit/roguetown/armor/leather/studded)
+
+/datum/supply_pack/rogue/bandit/Sawbones/leathercoatheavy
+	name = "heavy leather coat"
+	cost = 60
+	time_lock = 45 MINUTES
+	contains = list(/obj/item/clothing/suit/roguetown/armor/leathercoat/heavy)
+//medical
 
-//Misc
-/datum/supply_pack/rogue/bandit/Sawbones/leather/surgery_bag
+/datum/supply_pack/rogue/bandit/Sawbones/surgery_bag
 	name = "Surgery Bag"
-	cost = 120
+	cost = 30
 	contains = list(/obj/item/storage/fancy/skit)
 
-/datum/supply_pack/rogue/bandit/Sawbones/leather/patch_kit
+/datum/supply_pack/rogue/bandit/Sawbones/patch_kit
 	name = "personal patch kit"
-	cost = 120
+	cost = 30
 	contains = list(/obj/item/storage/fancy/ifak)
+//swords
+
+/datum/supply_pack/rogue/bandit/Sawbones/sword
+	name = "Iron Sword"
+	cost = 25
+	contains = list(/obj/item/rogueweapon/sword/iron)
+
+/datum/supply_pack/rogue/bandit/Sawbones/onehandedsword
+	name = "Short Sword"
+	cost = 25
+	contains = list(/obj/item/rogueweapon/sword/iron/short)
+
+/datum/supply_pack/rogue/bandit/Sawbones/onehandedMesser
+	name = "Messer"
+	cost = 25
+	contains = list(/obj/item/rogueweapon/sword/iron/messer)
+
+/datum/supply_pack/rogue/bandit/Sawbones/onehandedarmingsword
+	name = "Arming Sword"
+	cost = 35
+	time_lock = 45 MINUTES
+	contains = list(/obj/item/rogueweapon/sword/short)
+
+/datum/supply_pack/rogue/bandit/Sawbones/sword/steel
+	name = "Steel Sword"
+	cost = 35
+	time_lock = 45 MINUTES
+	contains = list(/obj/item/rogueweapon/sword)
 
-/datum/supply_pack/rogue/bandit/Sawbones/leather/gbottle
+/datum/supply_pack/rogue/bandit/Sawbones/swordgladius
+	name = "Gladius"
+	cost = 35
+	time_lock = 1.2 HOURS
+	contains = list(/obj/item/rogueweapon/sword/gladius)
+
+/datum/supply_pack/rogue/bandit/Sawbones/swordRapier
+	name = "Rapier"
+	cost = 35
+	time_lock = 1.2 HOURS
+	contains = list(/obj/item/rogueweapon/sword/rapier)
+//daggers
+
+/datum/supply_pack/rogue/bandit/Sawbones/dagger
+	name = "Iron Daggers"
+	cost = 25
+	contains = list(/obj/item/rogueweapon/huntingknife/idagger)
+
+/datum/supply_pack/rogue/bandit/Sawbones/daggerss
+	name = "Steel Dagger"
+	cost = 35
+	contains = list(/obj/item/rogueweapon/huntingknife/idagger/steel)
+
+/datum/supply_pack/rogue/bandit/Sawbones/combatdaggerss
+	name = "Combat Knife"
+	cost = 35
+	time_lock = 1.2 HOURS
+	contains = list(/obj/item/rogueweapon/huntingknife/cleaver/combat)
+
+//bottles
+
+/datum/supply_pack/rogue/bandit/Sawbones/gbottle
 	name = "Glass bottle"
-	cost = 24
+	cost = 5
 	contains = list(/obj/item/reagent_containers/glass/bottle)
 
-/datum/supply_pack/rogue/bandit/Sawbones/leather/botbomb
+/datum/supply_pack/rogue/bandit/Sawbones/botbomb
 	name = "Bottle bomb"
-	cost = 150
+	cost = 50
+	time_lock = 1.2 HOURS
 	contains = list(/obj/item/bomb)
 
-/datum/supply_pack/rogue/bandit/Sawbones/leather/poison
+/datum/supply_pack/rogue/bandit/Sawbones/poison
```

Comment:
This one needs to be removed. Since we're able to dip weapons into poison (daggers), Granting bandits access to a poison that will garuntee a kill after stabbing someone once is... very bad.